code_challenges.rb

```diff
@@ -18,31 +33,72 @@
 letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
 letter_a = 'a'
-# Expected output: ['tea', 'soda water']
-
-
-
-
-
-# -------------------3) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
-
+# Expected output: ['tea', 'water', 'soda water']
+#
+# #create a method that takes an array and a letter stored as an arguement
+# #use select on array to find if each value includes letter
+#
+def vowel_in_array (array, letter)
+  array.select do |value|
+    value.include? letter
+  end
+end
+p vowel_in_array(beverages_array, letter_o)
+p vowel_in_array(beverages_array, letter_a)
+#
+# # -------------------3) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
+#
 album1 = 'Rubber Soul'
 # Expected output: 'Rbbr Sl'
 album2 = 'Sgt Pepper'
 # Expected output: 'Sgt Pppr'
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
-
-
-
-
-
-# --------------------4) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a get_info method that returns a sentence with all the data from the bike object.
-
-# Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
-
-
-
+#
+# #create a method that takes a string as an arguement
+# #store vowels in variable
+# #use delete method to remove all vowels stored in variable
+#
+def no_vowels string
+  string.delete 'aeiouAEIOU'
+end
+p no_vowels album1
+p no_vowels album2
+p no_vowels album3
+#
+#
+# # --------------------4) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a get_info method that returns a sentence with all the data from the bike object.
+#
+# # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
+#
+class Bike
+  def initialize(model, wheels, current_speed)
```

Comment:
```suggestion
  def initialize(model)
```
the following parameters aren't needed since they're not being reference in the initialize method